blotter-template/blade/DataSource/bin/start-jar.sh

```diff
@@ -28,6 +28,12 @@ if [ $confreading = 1 ]; then
    java -jar "$jar" "$@"
    exit $?
 else
-   java -cp "$classpath" -jar "$jar" "$@" 2> "$LOGDIR"/java-$BLADENAME.log >/dev/null &
```

Comment:
It's a pity this file has so many copies. I know the jar name is different in each case but is there any way to centralise the common bit ?